README.md

```diff
@@ -2,6 +2,11 @@
 [![Build Status](https://platform-ci.scala-lang.org/api/badges/scalameta/scalameta/status.svg)](https://platform-ci.scala-lang.org/scalameta/scalameta)
 [![Join the chat at https://gitter.im/scalameta/scalameta](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/scalameta/scalameta?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
+## Versions
+Release: ![Release](https://img.shields.io/maven-central/v/org.scalameta/scalameta_2.12.svg)
```

Comment:
Oh, I have not looked at the website for ages. I did not realise that was autogenerated. Ill close this PR.